packages/bot-web-ui/src/assets/sass/save-modal.scss

```diff
@@ -129,37 +128,11 @@ div.radio-group {
 
 .save-modal__wrapper {
     @include mobile {
-        position: fixed;
-        z-index: 6;
-        width: 100%;
-        left: 0px;
-        top: 80px;
-        background: var(--general-main-1);
-
-        & form {
-            height: 100%;
+        form {
             display: flex;
             flex-direction: column;
             justify-content: space-between;
-        }
-        & .dc-page-overlay {
-            margin-top: -40px;
-
-            &__header {
-                position: fixed;
-                top: 4rem;
-                width: 100%;
-                height: 4rem;
-            }
-            &__content {
-                margin-top: 40px;
-
-                & .modal__content {
-                    padding: 1.6rem;
-                    max-height: calc(100vh - 150px);
-                    overflow: overlay;
-                }
-            }
+            height: 100%;
         }
         & .dc-input {
             width: 100% !important;
```

Comment:
Is this still required?

Reply:
> Is this still required?

Yes, we do. because the form has display flex and if we don't have height the height is equal to the content height and then buttons don't fix at the end of the screen